tests/Aspire.NATS.Net.Tests/AspireNatsClientExtensionsTests.cs

```diff
@@ -1,18 +1,31 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Aspire.Components.Common.Tests;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
 using Microsoft.Extensions.Hosting;
 using NATS.Client.Core;
+using OpenTelemetry.Trace;
 using Xunit;
 
 namespace Aspire.NATS.Net.Tests;
 
-public class AspireNatsClientExtensionsTests
+public class AspireNatsClientExtensionsTests : IClassFixture<NatsContainerFixture>
```

Comment:
`tests/Aspire.NATS.Net.Tests/ConformanceTests.cs` should also use the fixture. With the same connection string:
```csharp
    private string ConnectionString => RequiresDockerTheoryAttribute.IsSupported
                                                ? _containerFixture.GetConnectionString()
                                                : "nats://apire-host:4222";
```

Reply:
With the fixture added there, the requiredlogcategories seems to be incorrect. These are the ones I see:
```
Category: Microsoft.Extensions.Hosting.Internal.ApplicationLifetime
Category: Microsoft.Extensions.Hosting.Internal.Host
Category: Microsoft.Hosting.Lifetime
Category: NATS.Client.Core.Commands.CommandWriter
Category: NATS.Client.Core.Internal.SubscriptionManager
Category: NATS.Client.Core.Internal.InboxSubBuilder
Category: NATS.Client.Core.NatsSubBase
Category: NATS.Client.Core.NatsConnection
```

@eerhardt should we be checking for `StartsWith` instead of the full string? like `NATS.Client` here.

Reply:
I prefer full string, to make sure we are aware of any changes.

Reply:
Just curious - isn't the intent here to check that messages from this namespace `NATS.Client.*` are being logged?

Reply:
The intent is to ensure the correct log categories are used.

Reply:
So given:
```
Category: NATS.Client.Core.Commands.CommandWriter
Category: NATS.Client.Core.Internal.SubscriptionManager
Category: NATS.Client.Core.Internal.InboxSubBuilder
Category: NATS.Client.Core.NatsSubBase
Category: NATS.Client.Core.NatsConnection
```

.. `RequiredLogCategories` should include all of them?

Reply:
Correct. Just like:

https://github.com/dotnet/aspire/blob/fc571c2d80da8e41675a4d3792857e2f3d57c55e/tests/Aspire.Npgsql.Tests/ConformanceTests.cs#L26-L34